src/Mobs/Path.cpp

```diff
@@ -44,6 +44,21 @@ cPath::cPath(
 	// TODO: if src not walkable OR dest not walkable, then abort.
 	// Borrow a new "isWalkable" from ProcessIfWalkable, make ProcessIfWalkable also call isWalkable
 
+	a_BoundingBoxWidth = 1;  // Until we improve physics, if ever.
```

Comment:
The physics engine seems to make the assumption that all mobs are 1 block wide, not adding this line would make the PathFinder and the physics go out of sync, and consequently mobs will be weird.  this line will be removed when/ the physics is fixed. It seems to cause to problems other than the fact that spiders can get in 1-block wide corridors.